src/views/LoginView.vue

```diff
@@ -367,5 +367,9 @@ export default {
   .video-container__wrapper {
     width: 85%;
   }
+
+  .discordButton{
+    white-space: normal;
+  }
```

Comment:
Is this intended to be inside the media query? Atm it only applies to small devices because it's in the media query. Would it make more sense to move it outside and apply this style for all devices?

Reply:
@itsalaidbacklife It was intended to be inside media query as the test is out of button in mobile view. Also if we will move that outside it will work the same.
Let me know if I have to make changes.